src/actions/indexing/aioseo-posts-import-action.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+namespace Yoast\WP\SEO\Actions\Indexing;
+
+use wpdb;
+use Yoast\WP\SEO\Models\Indexable;
+use Yoast\WP\SEO\Repositories\Indexable_Repository;
+
+/**
+ * General reindexing action for indexables.
+ *
+ * @phpcs:disable Yoast.NamingConventions.ObjectNameDepth.MaxExceeded
+ */
+class Aioseo_Posts_Import_Action implements Indexation_Action_Interface {
+
+	/**
+	 * The transient cache key.
+	 */
+	const UNINDEXED_COUNT_TRANSIENT = 'wpseo_total_unindexed_aioseo_posts';
+	const IMPORT_CURSOR_VALUE       = 'wpseo_aioseo_import_cursor';
+
+	/**
+	 * Represents the indexables repository.
+	 *
+	 * @var Indexable_Repository
+	 */
+	protected $indexable_repository;
+
+	/**
+	 * The WordPress database instance.
+	 *
+	 * @var wpdb
+	 */
+	protected $wpdb;
+
+	protected $aioseo_to_yoast_map = [
```

Comment:
I believe we should map to the `_postmeta` table, not the `_indexable` table. This is because indexables are built based on the data in the `_postmeta` table. We can map to the indexables now, but then the indexables could never be rebuilt in the future.

Reply:
Yeap, good point, we're taking care of that now 🎉